src/analyzing/Constant_propagation.ml

```diff
@@ -623,6 +623,20 @@ let add_special_constants env lang prog =
 (* Entry point *)
 (*****************************************************************************)
 
+type propagate_basic_visitor_funcs = {
+  visit_definition :
+    env * Iter_with_context.context -> AST_generic.definition -> unit;
+  visit_expr :
```

Comment:
This one you're not using it yet it seems, maybe it's better to drop it until we need it?

Reply:
Done.